hw/kdrive/src/kdrive.c

```diff
@@ -117,7 +117,14 @@ KdDisableScreen(ScreenPtr pScreen)
     KdDisableColormap(pScreen);
     if (!pScreenPriv->screen->dumb && pScreenPriv->card->cfuncs->disableAccel)
         (*pScreenPriv->card->cfuncs->disableAccel) (pScreen);
+    if (!pScreenPriv->screen->softCursor &&
```

Comment:
can we have them separated out into their own commits - along with those field's definitions ?